### ISSUE TITLE
Pandoc not available in RStudio Preview

### ISSUE THREAD

**Issue by imanuelcostigan** [rstudio/rstudio#3645]
### System details

    RStudio Edition :  Desktop
    RStudio Version : 1.2.1045
    OS Version      :  Windows 10
    R Version       :  3.5.1

### Steps to reproduce the problem

Try to knit an RMarkdown document. Get the following message:

```
Error: pandoc version 1.12.3 or higher is required and was not found (see the help page ?rmarkdown::pandoc_available).
```

### Describe the problem in detail

On inspection, pandoc does not seem to be where RStudio thinks it should be:

```R
list.files(Sys.getenv("RSTUDIO_PANDOC"))
#> character(0)
```

I have not changed the `RSTUDIO_PANDOC` environment variable.

### Describe the behavior you expected

The Rmd file should knit and produce a HTML document

**Comment by andresrcs** [comment]
I had the same issue with build 1.2.1045 but now is working fine in build 1.2.1046

**Comment by dfalty** [comment]
Verified fixed on version 1.2.1090.

**Comment by ries9112** [comment]
I was having an issue related to this when creating a bookdown environment using GitHub actions and thought I would share what ended up working for me here:
![image](https://user-images.githubusercontent.com/32306488/75176696-de01fa00-56f1-11ea-8d0f-2d41a5532642.png)

I was getting the error:  

> **Error: pandoc version 1.12.3 or higher is required and was not found (see the help page ?rmarkdown::pandoc_available).**

What helped me was looking at another user's github project, so the code below belongs to user EmilHvitfeldt: https://github.com/EmilHvitfeldt/bookdown-github-actions-netlify/blob/master/.github/workflows/bookdown.yaml

![image](https://user-images.githubusercontent.com/32306488/75176116-c8400500-56f0-11ea-8386-6c3be5709315.png)

What worked for me was adding the step shown above in the steps for your github action job .yaml file before running the render_book() command. Note that `brew` would only work when running the job on a macOS configuration.

Sorry if this is unrelated, but this is what came up for me when I googled it and I could not find a solution related to my issue so I thought I would add to this thread.
